### ISSUE TITLE
Cannot authorize Google (Nest) account: Error 400: invalid_request

### ISSUE THREAD

**Issue by bruindustin** [iMicknl/ha-nest-protect#368]
### The problem

I'm a new user trying to configure my install of the Nest Protect integration (after having downloaded, rebooted, and begin the "add integration" workflow). I was rather immediately blocked when clicking the "Authorize your account" button.
![Screenshot 2024-10-16 at 23 42 13](https://github.com/user-attachments/assets/ef34b542-a053-4d19-b4af-fdc91bfb4ec5)

I received this error:
<img width="559" alt="Screenshot 2024-10-16 at 23 38 17" src="https://github.com/user-attachments/assets/1c4c545a-896f-4a4f-a67b-fd6b3e570e45">

which links to this page: https://developers.google.com/identity/protocols/oauth2/resources/oob-migration

### What version of this integration (ha-nest-protect) has the issue?

0.3.12

### What version of Home Assistant Core has the issue?

2024.10.2

### Device / Model

n/a

### Diagnostics information

n/a -  I don't believe this is possible until after we fix the auth issue

### Home Assistant log

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks (`)
```
</details>

**Comment by iMicknl** [comment]
Duplicate of #105, TLDR use latest beta.

**Comment by DaveTSG** [comment]
> TLDR use latest beta.

Tried that with latest beta, i.e. v0.4.0b7. Got the same error.

**Comment by iMicknl** [comment]
@DaveTSG than something went wrong with the installation, the beta uses a different auth method, thus you can never receive this error.
